server/konnectors/podcast.js

```diff
@@ -242,9 +242,11 @@ function createFileIfNotPresent (filename, path, url, callback) {
       callback(err)
```

Comment:
Can you also `return callback(err)` here ? 
This looks like the origin of the problem is here.